config/paths.js

```diff
@@ -1,13 +1,303 @@
 import { SCOPE_TYPE } from 'utils/constants';
+import { checkOwnership } from 'middlewares/custom';
+import { Stores } from 'models/stores';
+import { StoreProducts } from 'models/storeProducts';
+import { Suppliers } from 'models/suppliers';
+import { SupplierProducts } from 'models/supplierProducts';
+import config from 'config';
 export const paths = [
     {
-        path: '/assign-roles',
+        path: '/assign-roles/',
         scopes: [SCOPE_TYPE.SUPER_ADMIN],
         method: 'PUT'
     },
     {
-        path: '/roles',
-        scopes: [SCOPE_TYPE.ADMIN, SCOPE_TYPE.SUPER_ADMIN],
+        path: '/roles/',
```

Comment:
do we need all these trailing slashes here? Let's use an express plugin to remove trailing slashes if its just a "matching" thing